app/Controllers/Approval.php

```diff
@@ -0,0 +1,234 @@
+<?php
+
+namespace App\Controllers;
+
+use App\Models\AccountModel;
+use App\Models\EventModel;
+use App\Models\SocialMediaLinkModel;
+use App\Models\SpeakerModel;
+use App\Models\TeamMemberModel;
+
+enum Role: string
+{
+    case SPEAKER = 'speaker';
+    case TEAM_MEMBER = 'reviewer';
```

Comment:
This was a leftover of AI generated code 🙈 I removed the enum values alltogether since I don't use them anyway (i.e. I converted the backed enum into a regular enum).